pandas/core/arrays/string_arrow.py

```diff
@@ -4,6 +4,7 @@
 import re
 from typing import (
     TYPE_CHECKING,
+    Never,
```

Comment:
This is probably too new? (some CI is failing on this import)

Reply:
Thanks!

Reply:
Still gives some typing error ...

```
pandas/core/arrays/string_arrow.py:484: error: Return type "None" of "__pos__" incompatible with
return type "ArrowStringArray" in supertype "ArrowExtensionArray"  [override]
```

Reply:
It seems others are typed as `def __pos__(self) -> Self:`